packages/advjs/src/client/components/adv/AdvBlack.vue

```diff
@@ -76,7 +74,7 @@ watch(() => props.content, () => {
 <style lang="scss">
 .adv-black {
   background-color: rgba(0, 0, 0, 0.8);
-
+  top:0;
```

Comment:
Is there any problem without it?

Reply:
Yes. Otherwise you cannot see it.

Reply:
I know. The canvas does not have absolute. So if you add it, you'll not see the adv-black.